src/components/Login/Login.js

```diff
@@ -60,11 +60,14 @@ const Login = ({ onLogin }) => {
             value={password}
             onChange={(e) => setPassword(e.target.value)}
           />
-          <button className="login-button" type="submit">
-            Login
-          </button>
+          <button className="standard-btn" type="submit">LOGIN</button>
           {loginError && <p className="error-message">{loginError}</p>}
         </form>
+        <div className='demo-user-container'>
+          <button className="standard-btn" >DEMO NO USER</button>
+          <button className="standard-btn" >DEMO USER 1</button>
+          <button className="standard-btn" >DEMO USER 2</button>
+        </div>
```

Comment:
So are we using these buttons for showcasing the App with/without a login user-flow of typing?

Reply:
It's for demo purposes. you can also just manually visit the `/` path and you won't be logged in